Specification/8._Hardware_design_recommendations.md

```diff
@@ -4,22 +4,22 @@ permalink: /Specification/8._Hardware_design_recommendations/
 
 # Hardware design recommendations
 
-The points concerning hardware design practices are listed in this chapter.
-Following them is highly recommended, because that would ensure the highest level of inter-vendor compatibility
-and would allow the developers to avoid some of the common pitfalls.
+This chapter contains best-practice recommendations for hardware design.
+
+Following these guidelines will ensure the highest level of inter-vendor compatibility
+and allow the developers to avoid many common design pitfalls.
 
 ## Physical connector definition
 
+**Do not deviate from the recommendations provided in this section, as this may impact cross-vendor compatibility.**
+
 {% include lightbox.html url="/Specification/figures/connector_size_comparison.jpg" title="Size comparison of different UAVCAN connector types. The blue PCB is a small UAV avionic device given for scale." thumbnail=true %}
 
-The UAVCAN standard defines several connector types targeted for different application domains:
+The UAVCAN standard defines several connector types, targeted towards different application domains:
 from highly compact systems to large deployments, from low-cost to safety-critical applications.
-It is extremely undesirable to deviate from the recommendations provided in this section,
-because that could hinder the cross-vendor compatibility.
 
-The following table provides a summarized overview of all of the defined connector types.
-They are reviewed in more detail later.
-The image on the right comparatively demonstrates the size of the defined connector types.
+The following table provides a summarizes all of the currently defined connector types.
+The image to the right comparatively demonstrates the size of the connector types.
```

Comment:
I assume the images don't support captions? IF they do, I would make  this sentence into a caption (with appropriate changes).

Reply:
Sadly, our current layout does not support captions. It could be extended though.

> The following table provides a summarizes all of the currently defined connector types.

The sentence doesn't look correct.

Reply:
The following table provides a summary of all currently defined connector types.

Reply:
It isn't correct. Fixed. Thank you.

Reply:
@EShamaev I actually changed to "The following table provides an overview of the currently defined connector types."